implementation/gate/src/test/java/eu/efti/eftigate/controller/ConsignmentControllerTest.java

```diff
@@ -30,7 +30,7 @@
 @WebMvcTest(IdentifiersController.class)
 @ContextConfiguration(classes= {IdentifiersController.class})
 @ExtendWith(SpringExtension.class)
-class IdentifiersControllerTest {
```

Comment:
I think that renaming was not wanted , the actual controller did not change name

Reply:
Reverted this in the next commit